app/views/comments/_comments.html.erb

```diff
@@ -12,7 +12,7 @@
       <% end %>
     </tr>
 
-  <% @comments.each do |comment| %>
+  <% comments.each do |comment| %>
```

Comment:
@jywarren I adjusted user_controller.rb - https://github.com/publiclab/plots2/pull/4740/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R287 to update with this change so working everything working. Do you remember, if we use this partial _comments anywhere else except profile page and /comments? Let me know, and I will make the change accordingly. Thanks!

Reply:
I don't believe we use it anywhere else! So then this should be ready to merge? 👍

Reply:
yes, the work is complete according to me. you can merge it, if you find all changes good in review. Also, should I test on `unstable` once? Thanks!